src/AsmResolver.DotNet/Code/Cil/CilInstructionExtensions.cs

```diff
@@ -23,8 +24,8 @@ public static class CilInstructionExtensions
         public static int GetStackPopCount(this CilInstruction instruction, CilMethodBody? parent)
         {
             return GetStackPopCount(instruction,
-                parent is null
-                || !(parent.Owner.Signature?.ReturnsValue).GetValueOrDefault());
+                parent == null
+                || HasReturnValue(parent.Owner.Signature));
```

Comment:
Possible NRE when the `Signature` property is `null` (as a result of invalid metadata).